autotest.pm

```diff
@@ -312,10 +312,9 @@ sub start_process () {
 
             $0 = "$0: autotest";
             my $line = <$isotovideo>;
-            if (!$line) {
-                _exit(0);    # uncoverable statement
-            }
-            bmwqemu::diag("GOT $line\n");
+            _exit(0) unless $line;    # uncoverable statement
+            chomp($line);
```

Comment:
Why chomp is necessary?

Reply:
Because $line itself can contain a new line as received from the backend. So this commit removes *at least* one newline, often even two ;)